finalize-plugins/shell-final.go

```diff
@@ -27,17 +30,24 @@ func PlugInfo() *C.char {
 	return C.CString(string(pluginjson))
 }
 
+// Provide the plugin scope as a bitmask
```

Comment:
I'm not sure if its important to specify that its provided as a bitmask, though i'm open to hear your reasoning if you think its important

Reply:
`as a bitmask` is no important information. I totally agree.